data/sql/updates/pending_db_world/rev_1684589406305000945.sql

```diff
@@ -0,0 +1,10 @@
+--
+
+UPDATE `creature_addon` SET `auras` = '18950' WHERE `guid` IN (135170, 135171, 135172, 135173, 135174, 135175, 135345, 135346, 135347, 135348, 135349, 135350, 135351, 135352, 135353, 135354, 135355, 135356, 135357, 135358, 135359, 135360, 135361, 135362);
+INSERT INTO `creature_addon` (`entry`, `path_id`, `mount`, `bytes1`, `bytes2`, `emote`, `visibilityDistanceType`, `auras`) VALUES
+(16408, 0, 0, 0, 0, 0, 0, '18950');
+INSERT INTO `creature_template_addon` (`entry`, `path_id`, `mount`, `bytes1`, `bytes2`, `emote`, `visibilityDistanceType`, `auras`) VALUES
+(16408, 0, 0, 0, 0, 0, 0, '18950');
```

Comment:
Both these INSERTS require a DELETE preceding it. And it looks like the first one is like I said below

```suggestion

DELETE FROM `creature_template_addon` WHERE `entry` = 16408;
INSERT INTO `creature_template_addon` (`entry`, `path_id`, `mount`, `bytes1`, `bytes2`, `emote`, `visibilityDistanceType`, `auras`) VALUES
(16408, 0, 0, 0, 0, 0, 0, '18950');
```